packages/core/src/App/Components/Layout/Footer/help-centre.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Popover, Icon } from '@deriv/components';
+import { localize, getLanguage } from '@deriv/translations';
+
+const HelpCentre = () => {
+    const lang = getLanguage().toLowerCase();
+    const link_lang = lang === 'en' ? '' : `/${lang}`;
```

Comment:
should we have a `makeDerivStaticUrl` function in utils?

Reply:
sure, we can do that too 👍